### PR TITLE
vcmpbfp with Rc bit support

### PR DESCRIPTION
**Pull request by DrChat**
@benvanik You'll probably want to double check and make sure this works as intended.

### CODE REVIEW

File: src/xenia/cpu/frontend/ppc_emit_altivec.cc [hunk 1]
```diff
@@ -580,9 +580,10 @@ XEEMITTER(vcfpuxws128, VX128_3(6, 624), VX128_3)(PPCHIRBuilder& f,
 
 int InstrEmit_vcmpbfp_(PPCHIRBuilder& f, InstrData& i, uint32_t vd, uint32_t va,
                        uint32_t vb, uint32_t rc) {
+  // if vA or vB are NaN, the 2 high-order bits are set (0xC0000000)
   Value* va_value = f.LoadVR(va);
   Value* vb_value = f.LoadVR(vb);
-  Value* gt = f.VectorCompareSGT(va_value, vb_value, FLOAT32_TYPE);
+  Value* gt = f.VectorCompareSGE(va_value, vb_value, FLOAT32_TYPE);
```
Comment (benvanik) [inline src/xenia/cpu/frontend/ppc_emit_altivec.cc]: Change 'gt' to 'ge' now that it's GE.

**Comment by benvanik** [comment]
[qemu](http://copilotco.com/mail-archives/qemu.2009/msg02598.html) does an explicit check for NaN - do you think that's worth it, or does your comment mean this handles that?

**Comment by DrChat** [comment]
I changed the top comment to what it is because I'm pretty sure x86's VectorCompareXX will always return true on NaN checks, right?

**Comment by benvanik** [comment]
Probably :)

**Pull request merged**